packages/fxa-admin-panel/src/components/EmailBlocks/EmailBlock/index.tsx

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import './index.scss';
+
+const mockEmailBlock = {
+  account: {
+    uid: 'a1b2c3d4',
+    email: 'user@example.com',
+    createdAt: 1582246450000,
+    emailVerified: true,
+    emailBounces: [
+      {
+        email: 'user@example.com',
+        createdAt: 1582246455000,
+        bounceType: 'transient',
+        bounceSubType: 'suppressed',
+      },
+    ],
+  },
+};
+
+export const EmailBlock = () => {
+  const { account } = mockEmailBlock;
+  const accountIsVerified = account.emailVerified;
+  const emailBounce = account.emailBounces[0];
+  return (
+    <section className="email-block">
+      <ul>
+        <li className="flex justify-space-between">
+          <h3>{account.email}</h3>
+          <span className={`${accountIsVerified ? 'verify' : ''}`}>
+            {accountIsVerified ? 'verified' : 'not verified'}
+          </span>
+        </li>
+        <li className="flex justify-space-between">
```

Comment:
I'm curious about the use of utility classes here and concern classes elsewhere. I'm fine with either approach, mostly just curious if we should consider expanding in one direction or the other, _or_ perhaps have a utility library to serve as the foundation of everything and then build on top with individual concerns. Thoughts?

Reply:
Oh, I would also be satisifed with "this is just a foundation, don't pay too much attention to it yet" 😁

Reply:
Ideally we use the same methods/namings/approach across the payments server, content server, and admin panel, but what I have in this commit doesn't really yet have things laid out in a solid "preferred" way because I haven't seen any visual designs for the settings redesign yet (the blocker for the CSS architecture ADR I keep mentioning). I went with classes here that made sense for the admin panel and stuck some utility classes on components I'm hoping can be used in the settings redesign. The utility classes _may_ get pulled into their own file and expanded on for that redesign.

We do want to limit dependencies - I personally don't think we're going to need a CSS library for FxA, or if we do, we can make a tiny framework just for us, perhaps with these flex classes as a base for it. This admin panel project is going to end up staying pretty small, besides what I have here we're likely going to only have a few other components.

Reply:
Makes sense! Thanks.